packages/@sanity/cli/src/util/remoteTemplate.ts

```diff
@@ -318,3 +318,19 @@ export async function generateSanityApiReadToken(
     })
   return response.key
 }
+
+export async function setCorsOrigin(
+  origin: string,
+  projectId: string,
+  apiClient: CliApiClient,
+): Promise<void> {
+  try {
+    await apiClient({api: {projectId}}).request({
+      method: 'POST',
+      url: '/cors',
+      body: {origin: origin, allowCredentials: false},
+    })
+  } catch {
+    // Silent fail, it most likely means that the origin is already set
```

Comment:
It could be a good idea to [debug log the error using `debugIt`](https://github.com/sanity-io/sanity/blob/2bd96f628b170d80991faa3677eeaa67aab929b7/packages/@sanity/cli/src/debug.ts#L3) so we have the ability to investigate failures (e.g. by asking the user to repeat the command with debug flags enabled and share the logged error).

Reply:
💯 Totally!